apps/sms/views/conversation/js/startup.js

```diff
@@ -72,18 +68,17 @@
   exports.Startup = {
     init() {
       MessageManager.init();
-      Navigation.init();
       ConversationView.init();
-
-      ConversationView.once('visually-loaded', () => {
-        initLazyDependencies();
-        initHeaders();
-      });
-
       initShims().then(() => {
-        // Temporary workaround, navigation part will be revised in bug 1162030.
-        Navigation.defaultPanel = Navigation.getPanelName() || DEFAULT_PANEL;
-        Navigation.toDefaultPanel(Utils.params(exports.location.hash));
+        // We won't be able to access report and group panels directly because
+        // information.js is not loaded before ConversationView finishes
+        // loading.
+        Navigation.init();
+
+        ConversationView.once('visually-loaded', () => {
```

Comment:
not sure, I'll look closer; likely I just moved everything inside. The logic was that "Navigation.init" would provoke the rendering anyway, so this made quite sense to add the event handler just after this. But yeah, this is a event handler anyway :)